packages/docs/src/overwrite-components/head.astro

```diff
@@ -0,0 +1,11 @@
+---
+import type { Props } from "@astrojs/starlight/props";
+import Default from "@astrojs/starlight/components/Head.astro";
+---
+
+<Default {...Astro.props}>
+	<slot />
+</Default>
+<meta name="twitter:image" content="https://jimp-dev.github.io/jimp/share-banner.png">
```

Comment:
These have to be a static URL, otherwise some sites don't know how to resolve this quite right. If we add a custom domain in the future, we'll want to update this